components/Navigation/SectionNavigation.tsx

```diff
@@ -2,63 +2,83 @@ import Link from "next/link";
 import sections from "./sections/sections.json";
 import {
   Bullseye,
+  Card,
   Gallery,
   GalleryItem,
   Grid,
   GridItem,
-  Card,
+  Label,
   List,
   ListItem,
   CardBody,
   CardTitle,
+  Text,
+  TextContent,
+  TextVariants,
 } from "@patternfly/react-core";
 import { createUseStyles } from "react-jss";
+import classnames from "clsx";
+import CommandLineIcon from "./command-line";
+// import PenToolIcon from './pen-tool';
 
 const useStyles = createUseStyles({
-  description: {
-    color: "black",
-    fontSize: "var(--pf-global--FontSize--sm)",
+  gallery: {
+    columnCount: "auto",
+    columnWidth: "330px",
   },
-  cardBody: {
+  card: {
     overflow: "hidden",
-    height: 130,
+    breakInside: "avoid-column",
   },
-  link: {
-    textDecoration: "none",
-  },
-  card: {
-    height: 210,
-    textDecoration: "none",
-    letterSpacing: 2,
-    "& .pf-c-card__title": {
-      background: "var(--pf-global--BackgroundColor--dark-400)",
-      color: "var(--pf-global--Color--light-200)",
-      height: 80,
-    },
-    "&:hover": {
-      "& .pf-c-card__title": {
-        background: "var(--pf-global--BackgroundColor--dark-200)",
-      },
-    },
+  label: {
+    padding:
+      "var(--pf-global--spacer--sm) 12px var(--pf-global--spacer--sm) 12px",
   },
 });
 
 const SectionNavigation = () => {
   const classes = useStyles();
+
   return (
-    <Gallery hasGutter>
+    <Gallery
+      className={classnames(classes.gallery, "pf-u-display-block")}
+      hasGutter
+    >
       {Object.entries(sections).map(([parentKey, { title, items }]) => (
         <GalleryItem key={parentKey}>
-          <Card key={parentKey} className={classes.card} isSelectable>
-            <CardTitle className="pf-u-pb-lg">{title}</CardTitle>
+          <Card
+            key={parentKey}
+            className={classnames(
+              classes.card,
+              "pf-u-display-block pf-u-mb-md pf-u-background-color-100"
+            )}
+            isSelectable
+          >
+            <CardTitle className="pf-u-pb-md">
+              {title}
+              <Label
+                color="orange"
+                className={classnames(classes.label, "pf-u-float-right")}
+              >
+                <CommandLineIcon />
+              </Label>
+              {/*<Label color="purple" className={classnames(classes.label, "pf-u-float-right")}><PenToolIcon/></Label>*/}
```

Comment:
Extra commented out label
```suggestion
```

Reply:
@karelhala - we have two label variants and icons that need to be used in the cards. I included the second for your convenience.